content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -620,16 +620,16 @@ This allows the cluster to repair accidental modifications, and helps to keep ro
 up-to-date as permissions and subjects change in new Kubernetes releases.
 
 To opt out of this reconciliation, set the `rbac.authorization.kubernetes.io/autoupdate`
-annotation on a default cluster role or rolebinding to `false`.
+annotation on a default cluster role or default cluster role binding to `false`.
```

Comment:
```suggestion
annotation on a default cluster role or default cluster RoleBinding to `false`.
```

IMO: Why did you change "rolebinding" to "role binding"? I think this refers to the RoleBinding object.